[FM v0.6.0]--Add docs for Window functions (#179)
streamnative/function-mesh-website: docs/functions/function-crd.md

| `statefulConfig.pulsar.javaProvider.className` | The Java class name of the state storage provider implementation. The class must implement the `org.apache.pulsar.functions.instance.state.StateStoreProvider` interface. If not, `org.apache.pulsar.functions.instance.state.BKStateStoreProviderImpl` will be used. |
| `statefulConfig.pulsar.javaProvider.config` | The configurations that are passed to the state storage provider. |

## Window function configurations

Function Mesh provides the following fields for window functions in the CRD definition.

| Field                           | Description                                                                                                                                                                 |
|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `actualWindowFunctionClassName` | Required. The runner class name of the implemented window function.                                                                                                         |

Reviewer: This field can be optional and the default value is the same as the `spec.className `

| `lateDataTopic`                 | Optional. The late data topic for the late tuple messages. The late data topic must be defined when specifying a timestamp extractor class (`timestampExtractorClassName`). |
| `maxLagMs`                      | Optional. The maximum lag duration (in milliseconds) of the window function. By default, it is set to 0.                                                                    |

> | `slidingIntervalCount`          | Optional. The number of messages which the window slides after.                                                                                                             |

Reviewer 2: ```suggestion
| `slidingIntervalCount`          | Optional. The number of messages before the window slides.                                                                                                             |
```
